pkgs/development/python-modules/django-user-agents/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, django
+, user-agents
+}:
+
+buildPythonPackage rec {
+  pname = "django-user_agents";
+  version = "0.4.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "051m98z8n6lh159sn9fpj51hnmycdvsl6yd0cxl0xqyf8qhsxa6d";
+  };
+
+  doCheck = false;
```

Comment:
please comment as to why they are disabled

If tests are not available, then please use `pythonImportsCheck` to import the most important modules. This isn't as good as unit tests, but will usually give a good indication of run-time errors. Please see [pythonImportsCheck documentation](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#using-pythonimportscheck) for more information.